base/sparse/sparsevector.jl

```diff
@@ -1118,13 +1109,13 @@ end
 
 ### Binary arithmetics: +, -, *
 
-for (vop, fun, mode) in [(:_vadd, :AddFun, 1),
-                         (:_vsub, :SubFun, 1),
-                         (:_vmul, :MulFun, 0)]
+for (vop, fun, mode) in [(:_vadd, :+, 1),
```

Comment:
What duplicate symbols? You mean the functions a few lines down can be merged with these?